c2corg_ui/templates/helpers/view.html

```diff
@@ -276,6 +280,14 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
             <button class="gray-btn btn show-more" translate>show more</button>
           </div>
         % endif
+        % if document['doctype'] == 'waypoints':
+          <div class="list-item vertical-align" protected-url-btn
+               url="${request.route_path('routes_add') + '#w=' + str(document['document_id'])}">
+            <a>
+              <button class="btn orange-btn" translate>add a new route to this waypoint</button>
```

Comment:
It would make sense but I wanted to be consistent with the "add a new outing to this document" button here https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/helpers/view.html#L348